src/pages/wellness/CampusWellnessHub.tsx

```
@@ -0,0 +1,1333 @@
+      score: newMoodScore,
+      activities: newMoodActivities,
+      journal: newMoodJournal,
+      counselorNotified: newMood === "crisis" || newMoodScore < 30,
+    };
+    setMoodEntries((prev) => [entry, ...prev]);
+    setModalOpen(false);
+    setNewMood("neutral");
+    setNewMoodScore(50);
+    setNewMoodJournal("");
+    setNewMoodActivities([]);
+    addToast("success", "Mood entry logged successfully" + (entry.counselorNotified ? " — counselor notified" : ""));
```

ESLint

[error] Replace `"success",·"Mood·entry·logged·successfully"·+·(entry.counselorNotified·?·"·—·counselor·notified"·:·"")` with `⏎······"success",⏎······"Mood·entry·logged·successfully"·+·(entry.counselorNotified·?·"·—·counselor·notified"·:·""),⏎····`

(prettier/prettier)